api/app/models/transcript.model.js

```diff
@@ -12,8 +12,12 @@ module.exports = (sequelize, Sequelize) => {
       unique: true,
       allowNull: false
     },
+    pr_url: { 
```

Comment:
i think a transcript can only have one PR url if am not mistaken. that's the rationale for putting the field on the transcript itself.

Reply:
yeah but some edge cases mean we will be reverting it to null. e.g the pr wasn't merged hence the transcript. So we'll revert it to null then change the transcript status to be queued. In this sense and claim duration expiration, transcripts can be dynamic while review is fixed (it is tied to a claim). I think it's better to be in review since it already holds similar status like `mergedAt`, `submittedAt` etc.

Reply:
agree with @Emmanuel-Develops on this. If a PR is submitted but not merged, then another PR by another author will need to be tracked.